x-pack/plugins/alerting/server/routes/rule/apis/aggregate/transforms/transform_aggregate_body_response/v1.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RewriteResponseCase } from '@kbn/actions-plugin/common';
+import { RuleAggregationFormattedResult } from '../../../../../../../common/routes/rule/apis/aggregate/types/latest';
+
+export const transformAggregateBodyResponse: RewriteResponseCase<
```

Comment:
Let's avoid the `RewriteResponseCase` since its a computed type and use the new types I talked about above:

```
export const transformAggregateBodyResponse = (
  result: RuleAggregationFormattedResult
): AggregateRulesResponseV1 => ({
  ...
});
```

Reply:
just to confirm, the API response uses snake case now?

Reply:
Correct, API response are snake case. The frontend can be more relaxed with the types, as long as we use the snake case types from the server on the frontend, we can use the TS helpers to automatically convert it camel case (Or just manually map it :smile: )

Reply:
Also it does feel weird to leak the types from the server application layer to the frontend if the goal is to have well defined "sections" to our architecture. With the consequence of potentially more duplication, trade offs haha